app/models/curator/controlled_terms/nomenclature.rb

```diff
@@ -13,5 +13,7 @@ class ControlledTerms::Nomenclature < ApplicationRecord
     attr_json :id_from_auth, :string
 
     validates :type, presence: true, inclusion: { in: ControlledTerms.nomenclature_types.collect { |type| "Curator::ControlledTerms::#{type}" } }
+
+    has_paper_trail
```

Comment:
Out of curiosity why did you leave this without options like `on: %{create update destroy)`?

Reply:
There are some default settings in `config/initializers/paper_trail.rb`. I only added the `on:` option when the desired events-that-trigger-versions differs from that default (mostly in Mappings).